bundle/tests/scorecard/config.yaml

```diff
@@ -0,0 +1,70 @@
+apiVersion: scorecard.operatorframework.io/v1alpha3
+kind: Configuration
```

Comment:
Will we try to use the scorecard tests for this operator?

Reply:
I just committed all the stuff which was generated because I wasn't sure if it would break anything else which is generated later using the `operator-sdk` command. I suppose we can delete these tests.